microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -704,12 +722,8 @@ If WIP is turned on for your version of Microsoft Edge, the browser collects eve
 <!-- ====================================================================== -->
 ## Thank you!
 
-Microsoft Edge is made possible by the [Chromium][ChromiumMain] open-source project and other open-source software.  To view all of the software credits, navigate to `edge://credits`.  [Google Chrome Privacy Whitepaper][GoogleChromePrivacyWhitepaper] was used as a source for gathering related information about the Chromium open source project.
-
 Microsoft Edge is made possible by the [Chromium][ChromiumMain] open-source project and other open-source software.  To view all of the software credits, navigate to `edge://credits`.  [Google Chrome Privacy Whitepaper][GoogleChromePrivacyWhitepaper] was used as a source for gathering related information about the Chromium open-source project.
```

Comment:
@dalenaMSFT - I removed the dup paragraph.  You pointed it out in a different PR which is currently open.  The dup paragraph came in through a previous PR that's closed/merged to "master" & "live".  The present PR uses the old square brackets convention for URLs (in both copies of the dup paragraph).

Reply:
@dalenaMSFT - please review & approve this PR.